ui/src/components/Collection/CollectionItem/index.tsx

```diff
@@ -20,7 +20,7 @@ const CollectionItem = (props: ICollectionItem) => {
           : {})}
       >
         {props.collection.media && props.collection.media.length > 1 ? (
-          <div className="z-1 absolute inset-0 flex flex-row overflow-hidden">
+          <div className="z-1 pointer-events-none absolute inset-0 flex flex-row overflow-hidden">
```

Comment:
This has a side effect of removing the pointer around the outside of  the box, but it's still clickable. You can just set the z-index to a negative value so the images go behind everything else, making it scrollable:
```suggestion
          <div className="z-[-100] absolute inset-0 flex flex-row overflow-hidden">
```

Reply:
Again, I am not sure how you see those things. Thanks!